docs/testing.md

```diff
@@ -0,0 +1,78 @@
+CRI-Containerd Testing Guide
+============================
+This document assumes you have already setup the development environment (go, git, `cri-contaienrd` repo etc.).  
+ 
+Before sending pull requests you should at least make sure your changes have passed code verification, unit test and CRI validation test.
```

Comment:
Before sending pull requests you should at least make sure your changes have passed verification, unit, and CRI validation tests.